Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.Util.cpp

```diff
@@ -28,7 +28,7 @@ int CLuaFunctionDefs::GetValidPedModels(lua_State* luaVM)
 
     // Gather our custom skin model IDs allocated with engineRequestModel
     // (there might be some < 313 as well, and since we don't want duplicates, we start at 313, others are already included by the loop above)
-    for (const CClientModel* model : m_pManager->GetModelManager()->GetModelsByType(eClientModelType::PED, 313)) 
+    for (auto model : m_pManager->GetModelManager()->GetModelsByType(eClientModelType::PED, 313)) 
```

Comment:
`const auto&`, again, no need to copy the shared ptr, as its locked already by the vector.